Tests/MessageTests.swift

```diff
@@ -1,17 +1,16 @@
-import XCTest
-import UIKit
-
+import CoreLocation
 @testable import Daylight
+import UIKit
+import XCTest
 
-class MessageTests: XCTestCase {
-
-    func testKind() {
-        XCTAssertEqual(Message.Kind.longerMoreThanAMinute, Message.Kind(daylightLengthDifference: 70))
-        XCTAssertEqual(Message.Kind.longerLessThanAMinute, Message.Kind(daylightLengthDifference: 40))
-        XCTAssertEqual(Message.Kind.shorterMoreThanAMinute, Message.Kind(daylightLengthDifference: -120))
-        XCTAssertEqual(Message.Kind.shorterLessThanAMinute, Message.Kind(daylightLengthDifference: -40))
+extension Location {
+    static var testLocation: CLLocationCoordinate2D {
+        // Bonn
+        return CLLocationCoordinate2D(latitude: 50.73390258, longitude: 7.09812965)
```

Comment:
Future enhancement if you're bored: I've generally found that when testing it's always good to have one Northern/Western hemisphere and one Southern/Eastern hemisphere test to make sure stuff's not accidentally getting screwed up by sign changes on the lat/long 

Definitely not necessary to merge this though